src/libraries/System.Speech/src/System.Speech.csproj

```diff
@@ -0,0 +1,258 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <TargetFrameworks>$(NetCoreAppCurrent)-windows;netcoreapp2.0-windows;netstandard2.0</TargetFrameworks>
```

Comment:
Maybe that was already discussed, but why are we support netcoreapp2.0 which will be out of support when we are shipping NET6? For the lower .NETCoreApp version to be supported, I would have expect netcoreapp3.1 to be chosen.

Reply:
We will backport this and ship it in Jan. Not sure about netcoreapp2.0 vs 2.1

Reply:
As .NET Core 2.0 is out of support I would target 2.1. And after you backported the library we might want to drop the 2.1 asset then and update to 3.1. Presumably that should then be done across the board for all packages. Ad discussed in the nuget package assets meeting, in cases where we create holes (i.e. netstandard2.0 ref assembly which supports restore and building on .NET Core 2.x but during runtime the app will throw PNSE) we should add a target that errors out in case the platform isn't in support anymore. I will open an issue for the 2.x to 3.x upgrade.

Reply:
https://github.com/dotnet/runtime/issues/46400

Reply:
Changed to netcoreapp2.1 although maybe @ericstj had reasons for 2.0.

Reply:
Likely because it creates a hole in the support matrix as netcoreapp2.0 also supports netstandard2.0, but I'm not entirely sure.